docs/gsoc.rst

```diff
@@ -0,0 +1,100 @@
+Google Summer of Code
+=====================
+
+Read the Docs is hoping to participate in the Google Summer of Code in 2018.
+This page will contain all the information for students and anyone else interested in helping.
+
+Required Skills
+---------------
+
+Incoming students will need the following skills:
+
+* Intermediate python programming
```

Comment:
I think `django` and `docker` should be added

Reply:
Agreed re: Django -- there are projects that can be done without docker, so I'm less worried about that.